src/app/settings/settings.module.ts

```diff
@@ -40,7 +43,8 @@ import { RouterModule } from '@angular/router';
         AnimatedCheckmarkModule
     ],
     providers: [
-        SettingsService
+        SettingsService,
+        PrintOptionsMenuService
```

Comment:
Remove PrintOptionsMenuService and the import. Already part of another provider array. See other comment on print-settings.comp.ts